pkg/config/config.go

```diff
@@ -0,0 +1,36 @@
+package config
+
+import (
+	"context"
+	"errors"
+
+	"github.com/joho/godotenv"
+	"github.com/sethvargo/go-envconfig"
+)
+
+type Config struct {
+	WebDir string `env:"TODO_WEB_DIR,default=./web"`
+	Port   string `env:"TODO_PORT,default=:7540"`
+	DbFile string `env:"TODO_DBFILE,default=./pkg/db/scheduler.db"`
+}
+
+func NewConfig() (*Config, error) {
+	// Загружаем переменные окружения из .env файла
+	if err := godotenv.Load(".env"); err != nil {
+		err = errors.New("warning: .env file not found or couldn't be loaded")
+		return nil, err
```

Comment:
Теряешь сообщение с ошибкой. Также нет смысла создавать дополнительную переменную.
```suggestion
		return nil, fmt.Errorf("cannot load .env file: %w", err)
```